recipe/build.sh

```diff
@@ -24,16 +24,15 @@ export MPIF77=${COMPILER_PREFIX}/mpifort
 export MPIF90=${COMPILER_PREFIX}/mpifort
 
 ./configure --prefix=${PREFIX} \
-    --with-mpi=${PREFIX} \
-    --enable-shared=yes \
-    --enable-static=no
+            --with-mpi=${PREFIX} \
+            --enable-shared=yes \
+            --enable-static=no
 
-make
+make -j"${CPU_COUNT}"
 
-if [[ "${CONDA_BUILD_CROSS_COMPILATION:-}" != "1" || "${CROSSCOMPILING_EMULATOR}" != "" ]]; then
-    # MPI tests aren't working in CI (not uncommon)
+if [[ "$target_platform" == "linux-64" ]]; then
```

Comment:
@j34ni, I'm good with nearly all of your changes.  I'm wondering in this case why you've disabled osx-64.  I think it's fair to disable testing for all the cross-compiled and emulated OSes but I think we want to keep the native ones.

Reply:
@xylar I have not sorted out MVAPICH for `osx-`  (and `win`)  yet, and focused on `linux-` so far, to get something started (and since this is where *netmods* are most relevant)
Some help for these platforms would be much appreciated

Reply:
@j34ni, I'm not worried about mvapich not supporting these other OSes yet.  You're skipping those builds anyway.  But we want to test osx for mpich and openmpi as we were before.

Reply:
```suggestion
# Skip tests if cross-compiling or emulating
if [ "${CONDA_BUILD_CROSS_COMPILATION}" != "1" ]; then
```